### PR TITLE
Users/arjgupta/remove preview in name for rm tasks

### PR DESCRIPTION
**Pull request by arjgupta**

### CODE REVIEW

File: Tasks/Common/nuget-task-common/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -14,4 +14,4 @@
   "loc.messages.Error_NuGetToolInstallerFailer": "Tool install failed: %s",
   "loc.messages.Warning_UpdatingNuGetVersion": "Updating version of NuGet.exe to %s from %s. Behavior changes or breaking changes might occur as NuGet updates to a new version. If this is not desired, uncheck the 'Check for Latest Version' option in the task.",
   "loc.messages.Error_NoMatchingFilesFoundForPattern": "No matching files were found with search pattern: %s"
```
Comment (chshrikh) [inline Tasks/Common/nuget-task-common/Strings/resources.resjson/en-US/resources.resjson]: revert this change

File: Tasks/DotNetCoreCLI/Strings/resources.resjson/en-US/resources.resjson [hunk 2]
```diff
@@ -96,4 +96,4 @@
   "loc.messages.noPublishFolderFoundToZip": "A publish folder could not be found to zip for project file: %s.",
   "loc.messages.noWebProjctFound": "No web project was found in the repository. Web projects are identified by presence of either a web.config file or wwwroot folder in the directory.",
   "loc.messages.zipFailed": "Zip failed with error: %s"
-}
```
Comment (chshrikh) [inline Tasks/DotNetCoreCLI/Strings/resources.resjson/en-US/resources.resjson]: revert this unwanted change

File: Tasks/AzureVmssDeployment/task.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 0,
         "Minor": 0,
-        "Patch": 7
+        "Patch": 8
```
Comment (bishal-pdMSFT) [inline Tasks/AzureVmssDeployment/task.json]: I hope that as you mentioned, you will port these changes to releases/m123 and master as well and ensure that the versions there are greater than this version.

File: Tasks/DotNetCoreCLI/task.loc.json [hunk 4]
```diff
@@ -14,11 +14,10 @@
     "Agent"
   ],
   "demands": [],
-  "preview": true,
   "version": {
     "Major": 2,
     "Minor": 1,
-    "Patch": 0
+    "Patch": 1
```
Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreCLI/task.loc.json]: Please bump up version for DotNetCoreInstaller task as well. I had removed preview flag from there earlier, but did not bump up patch version. Now since we are bumping up for all tasks, please do so for DotNetCoreInstaller as well.

Comment (arjgupta) [reply Tasks/DotNetCoreCLI/task.loc.json]: done

File: Tasks/DotNetCoreCLI/task.json [hunk 5]
```diff
@@ -14,11 +14,10 @@
         "Agent"
     ],
     "demands": [],
-    "preview": true,
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/task.json]: This one is not required

Comment (arjgupta) [reply Tasks/DotNetCoreCLI/task.json]: have added it back preview=true and reverted patch version update

File: Tasks/AzureVmssDeployment/task.json (hunk 3 above)
Comment (arjgupta) [reply Tasks/AzureVmssDeployment/task.json]: yes i will do that
